conf/local.conf.sample

```diff
@@ -94,8 +105,22 @@ EXTRA_IMAGE_FEATURES = ""
 # certificate location for signing bundles
 #CERT_PATH = "${TOPDIR}/../sources/meta-yogurt/openssl-ca"
 
+WKS_FILES_mx8m_update = "imx8mm-connagtive-sdimage.wks"
```

Comment:
It is a little bit unfortunate that we have to add this variable to the local.conf again but I do not see a different way to solve this. The only way would be to create a new machine, as this is the place where the WKS_FILES variable is defined, but I would rather not do that as it does not any value. So I approve your solution.